CHANGES.md

```diff
@@ -1,3 +1,24 @@
+## edge-19.5.2
+
+* CLI
+  * Fixed `linkerd check` and `linkerd dashboard` failing when any control plane
+    pod is not ready, even when multiple replicas exist (as in HA mode)
+* Controller
+  * Fixed control plane components failing on startup when the Kubernetes API
+    returns an `ErrGroupDiscoveryFailed`
+* Proxy
+  * Added a dispatch timeout that limits the amount of time a request can be
+    buffered in the proxy
+  * Removed the limit on the number of concurrently active service discovery
+    queries to the Destination service
+* Internal
```

Comment:
I'd prefer that we omit this section -- none of this is user-facing.

Reply:
We frequently just have a _Thanks ..._ at the bottom to call out contributions

Reply:
Okay, what do you think of 2dbc9fef3b7be911d7e46d570b21733c05ddec53?